src/layout/sidebar/Sidebar.tsx

```diff
@@ -8,19 +8,22 @@ interface optionsProps {
   route: string;
   isDisabled?: boolean;
   isHidden?: boolean;
+  additionalRoutes?: string[];
```

Comment:
I'd discourage bringing even more entities into our routing tbh.
I understand our current limitations, and with better setup we would have properly nested separated pages etc.

For now we can just check if `location.pathname` contains the `routeItem.route` string and drop the additional routes, wdyt